README.md

```diff
@@ -49,9 +49,9 @@ $ source <(summon --bash-completion-script `which summon`)
 ```
 
 After that, you can call `summon` with the required command. To create a
-project, use `summon new` command specifying the prefered CLI options, follow
+project, use `summon new` command specifying the preferred CLI options, follow
 the instructions during the interactive process of the project creation, and a
-new project would be created in a subfolder as well as a repository under your
+new project would be created in a sub folder as well as a repository under your
```

Comment:
Looks like this should be `subfolder` according to various dictionaries.
* https://www.merriam-webster.com/dictionary/subfolder
* https://en.oxforddictionaries.com/definition/subfolder

```suggestion
new project would be created in a subfolder as well as a repository under your
```